indico/htdocs/sass/modules/abstracts/_email_templates.scss

```diff
@@ -55,16 +55,65 @@
     }
 }
 
-.email-template-list {
-    list-style-type: none;
-    padding-left: 0;
-    width: 800px;
+#email-template-manager {
+    max-width: 800px;
+
+    .email-template-wrapper {
+        padding: 0 0 1em 1em;
+
+        .email-templates {
+            width: 800px;
+            max-height: 800px;
+            overflow: auto;
+
+            > ul {
+                list-style-type: none;
+                padding-left: 0;
+                padding-right: 20px;
+
+                > li {
+                    width: 775px;
+                }
+            }
+        }
+
+        .email-preview {
+            display: none;
+
+            .email-subject {
+                padding-bottom: 10px;
+            }
+
+            .email-body {
+                color: $light-black;
+            }
+        }
+
+        .regform-section-sortable-placeholder {
+            @extend .i-box;
+            @include border-all($gray, dashed);
+            @include box-shadow(none);
+
+            background-color: $light-gray;
+        }
+
+        .email-preview-btn {
+            color: $link;
```

Comment:
Not a strong opinion here. You need this because the `<a>` tag doesn't an `href="#"` attribute. If you add it you won't need to specify underline on hover either. OTOH, you will need to do `evt.preventDefault()` in the Javascript handling the click.